### PR TITLE
doc: fix github url

### PR DESCRIPTION
**Pull request by makelinux**
---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?

### CODE REVIEW

**Review by jcastill — APPROVED** [review]
lgtm

File: docs/index.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ To clone the current main (development) branch run:
 
 .. code::
 
-    git clone git://github.com/sosreport/sos.git
+    git clone git@github.com:sosreport/sos.git
```
Comment (arif-ali) [inline docs/index.rst:20]: would it be better to have the http endpoint, that way people don't have to specifically setup ssh? Especially if people don't have GH setup?

**Review by arif-ali — APPROVED** [review]
looks ok to me, just a comment

**Review by TurboTurtle — APPROVED** [review]

File: docs/index.rst (hunk 1 above)
Comment (TurboTurtle) [reply docs/index.rst:20]: IIRC doesn't GH have plans in play to drop non-authenticated http? Meaning someone would have to do some setup/configuration anyways (and GH strongly encourages the use of ssh)?

**Pull request merged**